packages/frontend/src/components/SimpleChart.tsx

```diff
@@ -83,6 +83,19 @@ export const SimpleChart: FC<SimpleChartProps> = ({
     chartType,
     chartConfig,
 }) => {
+    useEffect(() => {
+        if (!chartRef.current) return;
+
+        const listener = () => {
+            const eCharts = chartRef.current?.getEchartsInstance();
+            eCharts?.resize();
+        };
+
+        window.addEventListener('resize', listener);
+        /* eslint-disable-next-line */
```

Comment:
Why do we need to disable eslint here ? 
I suspect that's because `listener` might not be defined. this happens because if the `chartRef.current` is undefined we make an early return. 
What if we didn't have this logic at all ? we can create the event listener no matter what since we are removing it on "uumount"